src/olympia/activity/tests/test_constants.py

```diff
@@ -0,0 +1,52 @@
+from django.utils.translation import gettext_lazy as _
+
+import pytest
+
+from olympia.constants.activity import (
+    LOG,
+    LOG_HIDE_DEVELOPER,
+    LOG_OBSOLETE,
+    LOG_REVIEW_QUEUE_DEVELOPER,
+)
+
+
+def is_translated(value):
+    """Return True if `value` is a translated string. It simply looks at the
+    type: since translated strings for constants should be lazy, their type
+    should be a lazy proxy, not `str`."""
+    return not isinstance(value, str)
+
+
+def test_is_translated():
+    assert is_translated(_('Add-on'))
+    assert not is_translated('Add-on')
+
+
+@pytest.mark.parametrize(
+    'action',
+    [
+        action
+        for action in LOG
+        if action.id not in LOG_HIDE_DEVELOPER and action.id not in LOG_OBSOLETE
```

Comment:
Should we be excluding obsolete activities from this check?  Historic activities could still appear in existing views exposed to users.

Reply:
I know, it's not exactly great, but better than nothing. There are a number of existing obsolete ones that don't have translations, but I'm pretty sure have never been used. I wanted to at least ensure the ones we are currently using are translated.